ISSUE_TEMPLATE

```diff
@@ -1,10 +1,18 @@
 <!--Please use the template provided below when reporting bugs:-->
+## Link to bug demonstration repository.
+<!--
+A demo repository will show the combination of module versions being used,
+command-lines and configurations.  NYC is very complex, it is difficult or
+even impossible to triage most bugs without this information.
+
+If you cannot share a link to your repository (maybe it's private) then you must
+create a basic `nyc-bug-demo` repository, link to that.
```

Comment:
My only nit here is the extra comma, I would rewrite it as:

`...repository and link to that please.`